### ISSUE TITLE
The script using demo cannot be aligned with the result generated using webUI

### ISSUE THREAD

**Issue by Haowenyue** [tencent-ailab/IP-Adapter#318]
For the same input, the same seed, and the same model weights, the results we get with the ip-adapter-faceid inference script are quite different from the results we get with the webU.

**Comment by xiaohu2015** [comment]
yes, it may be different, webui and diffusers have some difference

**Comment by Haowenyue** [comment]
> yes, it may be different, webui and diffusers have some difference

I wonder if there is a way to align the results of the inference script with the results of the webUI, thanks!

**Comment by aiXia121** [comment]
> yes, it may be different, webui and diffusers have some difference
@xiaohu2015 
hi, thanks for your excellent work.
I have one question about using your xxx.ipynb example, i just copy all the code from ip_adapter_sdxl_demo.ipynb into one python ip_adapter_sdxl_demo.py , which just for running in remote device.
But, the results are different,  is there any idea for remodifying the results for the same?
![ip_sdxl](https://github.com/user-attachments/assets/b0ac8044-a4fc-42ce-90d3-0ccb0e3f2d75)
